src/WebJobs.Script.WebHost/Metrics/FlexConsumptionMetricsPublisher.cs

```diff
@@ -106,7 +111,8 @@ internal async Task OnPublishMetrics()
                         TotalTimeMS = (long)_intervalStopwatch.GetElapsedTime().TotalMilliseconds,
                         ExecutionCount = FunctionExecutionCount,
                         ExecutionTimeMS = FunctionExecutionTimeMS,
-                        IsAlwaysReady = IsAlwaysReady
+                        IsAlwaysReady = IsAlwaysReady,
+                        HostMetrics = _metricsProvider.GetHostMetricsOrNull()
```

Comment:
I'm confused about the structure here. All the existing flat record metrics are already "host metrics" - so why do we need this new "host metrics" nesting?

Reply:
I thought it would be cleaner to separate the ones we need for scaling vs the billing metrics

Reply:
I don't see why - these are all just host level metrics, to be used by consumers for whatever purposes they may need. Logically they all seem to be peers to the metrics we're already logging, so I say we just inline them.

Reply:
Sure. In that case can we move the `internal class Metrics` class to Core? HostMetricsProvider.GetHostMetricsOrNull would need to change in how we provide these values instead of creating HostMetricsPayload like we do now

Reply:
moved host metrics inline